tests/Neo.UnitTests/SmartContract/UT_InteropService.cs

```diff
@@ -300,6 +300,20 @@ public void TestRuntime_CheckWitness()
             action.Should().Throw<ArgumentException>();
         }
 
+        [TestMethod]
+        public void TestRuntime_CheckWitness_Null_ScriptContainer()
+        {
+            byte[] privateKey = { 0x01,0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01,
+                0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01, 0x01};
+            KeyPair keyPair = new(privateKey);
+            ECPoint pubkey = keyPair.PublicKey;
+
+            var engine = GetEngine();
+
+            engine.CheckWitness(pubkey.EncodePoint(true)).Should().BeFalse();
+            engine.CheckWitness(pubkey.EncodePoint(true)).Should().BeFalse();
```

Comment:
Two checks?

Reply:
can we just leave it this way for now? My bad, did not notice it at all.

Reply:
Just fix in a new pr